libr/include/r_cmd.h

```diff
@@ -16,9 +16,11 @@ extern "C" {
 #define R_CMD_MAXLEN 4096
 
 typedef enum r_cmd_status_t {
-	R_CMD_STATUS_OK = 0,
-	R_CMD_STATUS_INVALID,
-	R_CMD_STATUS_EXIT
+	R_CMD_STATUS_OK = 0, // command handler exited in the right way
+	R_CMD_STATUS_WRONG_ARGS, // command handler could not handle the arguments passed to it
+	R_CMD_STATUS_ERROR, // command handler had issues while running (e.g. allocation error, etc.)
+	R_CMD_STATUS_INVALID, // command could not be found (e.g. it does not exist)
```

Comment:
Plenty of existing code although all from newshell that uses STATUS_INVALID as general purpose error not necessarily command could not be found.  It might be fine if the description gets updated. STATUS_INVALID meaning any shell level error command doesn't exist, bad expression in the `@` modifier and others.

Reply:
See also https://github.com/radareorg/radare2/issues/16147

Reply:
Please check now, I have improved the comment, should be good.

Reply:
Looks good.